libyul/optimiser/ControlFlowSimplifier.cpp

```diff
@@ -180,7 +180,8 @@ void ControlFlowSimplifier::visit(Statement& _st)
 
 void ControlFlowSimplifier::simplify(std::vector<yul::Statement>& _statements)
 {
-	GenericFallbackReturnsVisitor<OptionalStatements, If, Switch> const visitor(
+	GenericVisitor visitor{
+		VisitorFallback<OptionalStatements>{},
```

Comment:
```suggestion
		[](auto&&) { return OptionalStatements{}; },
```
That's what I meant - we could just do this everywhere instead of defining ``VisitorFallback`` - whatever you think is more readable :-).